### ISSUE TITLE
Initial Exploration

### ISSUE THREAD

**Issue by jkuruzovich** [RPI-DATA/classification#1]
- [ ] Identify high quality notebooks in classification for business & eng.
- [ ] identify data to be used 
- [ ] create initial directory.

Some sites to look at for initial analysis:

DATA8 http://data8.org   https://github.com/data-8/materials-su18

Data100  http://www.ds100.org  https://github.com/DS-100/fa17

**Comment by jkuruzovich** [comment]
Figure out easy way to get multiple datasets.   
https://github.com/Kaggle/kaggle-api

**Comment by jkuruzovich** [comment]
https://github.com/amueller/scipy-2018-sklearn

**Comment by sunnyr3** [comment]
Types of classification algorithms in machine learning:
- [ ] Linear Classifiers: Logistic Regression, Naive Bayes Classifier 
- [ ] Support Vector Machines
- [ ] Decision Trees
- [ ] Boosted Trees
- [ ] Random Forest
- [ ] Neural Networks
- [ ] Nearest Neighbor

**Comment by sunnyr3** [comment]
Sample analysis using logistic regression:
[https://github.com/SidSachdev/AdvertisingLogisticRegresssion](https://github.com/SidSachdev/AdvertisingLogisticRegresssion)

**Comment by sunnyr3** [comment]
An article that simply summarizes each classification algorithm:
[Types of classification algorithms in Machine Learning](https://medium.com/@sifium/machine-learning-types-of-classification-9497bd4f2e14) by Mandeep Sidana

**Comment by jkuruzovich** [comment]
https://github.com/amueller/scipy-2018-sklearn

**Comment by sunnyr3** [comment]
Example notebook for Random Forest:
[reproduce-stock-market-direction-random-forests](https://github.com/jmartinezheras/reproduce-stock-market-direction-random-forests/blob/master/2016_StockDirection_RF.ipynb)

**Comment by JacksonFWong** [comment]
Notebook that explores classification using a census dataset

Performs:

- Feature Exploration (Uni and Bi-variate)
- Feature Imputation
- Feature Selection
- Feature Encoding
- Feature Ranking
- Machine Learning with sklearn and Tensorflow
- Random Search
- Accuracy, Precision, Recall, and f1 calculations
- ROC Curve

Link: https://github.com/dformoso/sklearn-classification.git

Also uploaded to notebooks folder
